src/snackbar/snackbar-container.svelte

```diff
@@ -6,8 +6,11 @@
   export let position;
   let registeredSnackbars = new Set();
 
-  function removeSnackbar(key) {
+  function removeSnackbar(key, closedEarly) {
     registeredSnackbars.delete(key);
+    if (key.resolveExpiredPromise != null) {
```

Comment:
I found that when checking for the "nullness" of a function, it is more common to do it as such
```suggestion
    if (typeof key.resolveExpiredPromise === 'function') {
```
to avoid more possible errors at once